src/Dfe.PlanTech.Domain/CategorySection/CategorySectionDto.cs

```diff
@@ -43,7 +43,8 @@ public CategorySectionDto(string? slug, string name, bool retrievalError, Sectio
     {
         if (date == null)
             return null;
-        var localTime = date.Value.ToLocalTime();
+        var britishTimeZone = TimeZoneInfo.FindSystemTimeZoneById("GMT Standard Time");
```

Comment:
Would it be better to store it as UTC, then just display as the user's local time on the frontend?

Reply:
I'd gone this path so it would work with js disabled but maybe I could make this a fallback and have a JS time for the main use case

Reply:
Whatever you think is best mate! I just _generally_ would prefer to store times in UTC because I find that it's _generally_ much easier to manage that way. 

I hadn't considered the JS issue though, that's a good shout. I wonder if we could use Azure Frontdoor + [server variables](https://learn.microsoft.com/en-us/azure/frontdoor/rule-set-server-variables) to set the user's geo-location as a header? I've never tested this though.

Reply:
Resolving following huddle